toolkit/locales/en-US/chrome/global/aboutSupport.ftl

```diff
@@ -45,6 +65,13 @@ aboutSupport-appBasicsMultiProcessSupport = Multiprocess Windows
 aboutSupport-appBasicsSafeMode = Safe Mode
 aboutSupport-appBasicsProfiles = Profiles
 
+# This string displays the number of open multiprocess (e10s) windows compared
+# to the overall number of open windows, and the current status of e10s in the
+# browser.
+# $remote (Number) - Number of multiprocess windows
+# $total (Number)  - Total number of open windows
+# $status (String) - Current status of e10s
```

Comment:
Uhm, technically yes, `$status` is the array index. But the targets of these comments are localizers, not developers.
```
{ $remote }/{ $total } ({ $status })
```
The alternative would be to go into way too many details.
```
# $status (Number) - Numerical index for the current status of e10s. The corresponding status message will be displayed in the UI, e.g. “Enabled by user” for 0.
```